TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Alerts/AddAlert/Index.cshtml

```diff
@@ -0,0 +1,60 @@
+@page "/alerts/add"
+@model IndexModel
+@addTagHelper *, Joonasw.AspNetCore.SecurityHeaders
+@{
+    ViewBag.Title = "Add an alert";
+}
+
+@section Styles
+    {
+    <link rel="stylesheet" href="~/Styles/Components/accessible-autocomplete.min.css" asp-append-version="true" />
+}
+
+@section Scripts
+    {
+    <script src="~/Scripts/Components/accessible-autocomplete.min.js"></script>
+    <script asp-add-nonce="true">
+        window.onload = function () {
+            accessibleAutocomplete.enhanceSelectElement({
+                selectElement: document.querySelector('#@nameof(Model.AlertTypeId)')
+            })            
+        }
+    </script>
+}
+
+@section BeforeContent {
+    <govuk-back-link href="@LinkGenerator.PersonAlerts(Model.PersonId)">Back</govuk-back-link>
+}
+
+<h1 class="govuk-heading-l">@ViewBag.Title</h1>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <form method="post">
+            <govuk-select asp-for="AlertTypeId" label-class="govuk-label--m">
+                <govuk-select-item value=""></govuk-select-item>
+                @foreach (var alertType in Model.AlertTypes!)
+                {
+                    <govuk-select-item value="@alertType.AlertTypeId">@alertType.Name</govuk-select-item>
+                }
+            </govuk-select>
+
+            <govuk-textarea asp-for="Details" label-class="govuk-label--m">
+                <govuk-textarea-hint>You can enter up to 4000 characters</govuk-textarea-hint>
```

Comment:
This can go on the `Details` property on the `PageModel` (`[Display(Description = "You can enter...")]`

Reply:
Done